LocalSettings.php

```diff
@@ -5,22 +5,9 @@
 # Edit the CustomSettings.php file for your need.
 # All the configuration in this file configurable via environmental variable.
 
-# This debugging block was not added by default
-# it was included here, as version upgrades from 1.38 to 1.39 prooved 
-# to be very hard without error_reporting as the top line
-if (loadenv('APP_DEBUG', false) === "true") {
-  $hostName = gethostname();
-  error_reporting( -1 );
-  ini_set( 'display_errors', 1 );
-  // hostname will allow to create separate log file for each instance
-  $wgDebugLogFile = "/var/www/html/mediawiki-debug-$hostName.log";
-  $wgDebugComments = true;
-  $wgShowExceptionDetails = true;
-  $wgDebugDumpSql = true;
-  $wgDebugToolbar = true;
-  $wgShowDebug = true;
-  $wgDevelopmentWarnings = true;
-}
+#TODO: This is temporary config for 1.37 version
+error_reporting( 0 );
```

Comment:
@arulrajnet disabling error reporting was the reason the 1.39 upgrade related failures were not showing up. even if we enable it in customsettings.php, any error before the @include line at the bottom of this file, will go unreported. that's what made it difficult to debug yesterday. 

we can take this back to customsettings.php, but when you turn on app_debug = true, it will report only errors after customsettinngs, not before.